src/sql/SqlResult.ts

```diff
@@ -350,7 +362,6 @@ export class SqlResultImpl implements SqlResult {
 
     /**
      * Checks if there are rows to iterate in a recursive manner, similar to a non-blocking while block
-     * @internal
```

Comment:
I guess we had this before, therefore I added other internal labels. Let's delete this one too